### ISSUE TITLE
Url life time

### ISSUE THREAD

**Issue by Ptelea** [microsoftgraph/microsoft-graph-docs#8984]
The url parameter of the Thumbnail resource type seems to expire within 24 hours. It might be a good idea to mention this somewhere. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 34278783-5c78-7257-bb3b-469d0046743d
* Version Independent ID: 35192842-4fc6-b17d-d228-4e1edcfc5035
* Content: [Thumbnail - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/resources/thumbnail?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/resources/thumbnail.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/resources/thumbnail.md)
* Product: ****
* Technology: **microsoft-graph**
* GitHub Login: @JeremyKelley
* Microsoft Alias: **JeremyKelley**

**Comment by attakorns** [comment]
Hmm that's why my old URLs have stopped working! Did you notice that it lasts longer before? I remember working back in early 2020 and it was definitely lasting longer than 24 hrs.

Is there any alternative that you know of to create a non-expiring pre-authenticated URL? I need to be able to programmatically upload a file and send this non-expiring URL for someone else to view at a later date.

**Comment by r-a-y** [comment]
Yes, the expiry time should preferably also be added as a property to the JSON response.

This also applies for the Drive Item thumbnail endpoint as well:
https://docs.microsoft.com/en-us/graph/api/driveitem-list-thumbnails?view=graph-rest-1.0

Not sure if expiry time is only applicable to SharePoint and OneDrive for Business.
